ide/static/js/content.js

```diff
@@ -1299,9 +1339,11 @@ class Content extends React.Component {
           <div id="sidebar-scroll" className="col-md-12">
              <h5 className="sidebar-heading">ACTIONS</h5>
              <TopBar
+              isPublicSharing={this.state.isShared}
               exportNet={this.exportNet}
               importNet={this.importNet}
               saveDb={this.saveDb}
+              saveModel={this.saveModel}
```

Comment:
Change the method name to update model (if I understand the functionality correctly

Reply:
I have explained this function above, please have a look at it, it’s not just updating the already existing model, it updates the model only if it already exists else it saves the model.